### ISSUE TITLE
Add more tasks in parallel on sisyphus zimit worker

### ISSUE THREAD

**Issue by benoit74** [kiwix/operations#213]
We are currently running 3 tasks in parallel on zimit.kiwix.org worker sisyphus:

![image](https://github.com/kiwix/operations/assets/7102089/90637035-65f0-4460-b8e2-849166b8fc63)

Due to the significant pipe ahead of us and the fact that running only 3 tasks seems a bit conservative, we will try to run 4 tasks in parallel.

I will:
- change zimit task settings to use 1 CPU and 3840MiB of RAM instead of 2 CPUs and 5GiB RAM (to be done in zimit/ui/ui-configs.cm.yaml)
- update all existing requested tasks in DB accordingly (ongoing tasks will finish as they are)

**Comment by benoit74** [comment]
UI reconfigured (and restarted to apply new configuration). 

Manual test of new task creation is OK, appropriate settings are passed.

SQL statement used:

```
UPDATE requested_task
SET config =  jsonb_set(jsonb_set(jsonb_set(config::jsonb, '{resources, memory}', '4026531840'::jsonb), '{resources, shm}', '4026531840'::jsonb), '{resources, cpu}', '1'::jsonb)
WHERE config::jsonb -> 'image' ->> 'name' = 'ghcr.io/openzim/zimit';
```

Statement to confirm settings have been applied (this is visible in the farm UI as well)

```
SELECT id, config::jsonb -> 'resources' ->> 'cpu' AS cpu, config::jsonb -> 'resources' ->> 'memory' AS memory, config::jsonb -> 'resources' ->> 'shm' AS shm
FROM requested_task
WHERE config::jsonb -> 'image' ->> 'name' = 'ghcr.io/openzim/zimit';
```

Let's confirm later today and this week that everything is working as expected with 4 tasks in parallel and no major failure occured.
